amundsen_application/static/js/components/TableDetail/RequestMetadataForm/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import './styles.scss';
+
+import { sendNotification } from 'ducks/notification/api/v0';
+import { NotificationType } from 'interfaces/Notifications';
+import { AvatarLabelProps } from 'components/common/AvatarLabel';
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+
+interface StateFromProps {
+  userEmail: string;
+  tableName: string;
+  tableOwners: Array<String>;
+}
+
+type RequestMetadataProps = StateFromProps;
+
+interface RequestMetadataState {
+  isOpen: boolean,
+}
+
+export class RequestMetadataForm extends React.Component<RequestMetadataProps, RequestMetadataState> {
+  public static defaultProps: RequestMetadataProps = {
+    userEmail: '',
+    tableName: '',
+    tableOwners: [],
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: true,
```

Comment:
Looks like you're initializing to open by default for testing.

Consider that an external action is going to trigger this component to open and close. You will likely need to redux to change TableMetadata state and pass that value into this component in `mapStateToProps`

Reply:
^ yeah that would make sense. I'll address this in the follow up PR where I create the component to open this component